x-pack/plugins/threat_intelligence/cypress/screens/indicators.ts

```diff
@@ -23,7 +23,7 @@ export const FLYOUT_TITLE = `[data-test-subj="tiIndicatorFlyoutTitle"]`;
 
 export const FLYOUT_TABS = `[data-test-subj="tiIndicatorFlyoutTabs"]`;
 
-export const FLYOUT_TABLE = `[data-test-subj="tiFlyoutTableMemoryTable"]`;
+export const FLYOUT_TABLE = `[data-test-subj="tiFlyoutTableTabRow"]`;
```

Comment:
For some reason, this test subj has been altered. Investigation ticket: https://github.com/elastic/security-team/issues/5019